sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Latest.java

```diff
@@ -153,11 +153,12 @@ public T extractOutput(TimestampedValue<T> accumulator) {
    * @param <K> The key type of the elements being combined.
    * @param <V> The value type of the elements being combined.
    */
-  public static <K, V> PTransform<PCollection<KV<K, V>>, PCollection<KV<K, V>>> perKey() {
+  public static <K, V> PerKey<K, V> perKey() {
     return new PerKey<>();
   }
 
-  private static class Globally<T> extends PTransform<PCollection<T>, PCollection<T>> {
+  /** Implementation of {@link #globally()}. */
+  public static class Globally<T> extends PTransform<PCollection<T>, PCollection<T>> {
```

Comment:
I started a thread about this on beam-dev https://lists.apache.org/thread.html/f448eb9f531bec2c0cd70e790dc8d2198d8a1bf9d2b85f94d9ac904a@%3Cdev.beam.apache.org%3E . There indeed is no section about that, but perhaps my arguments in the thread shall be convincing.